.golangci.yml

```diff
@@ -23,11 +23,10 @@ linters:
   - nolintlint
   - prealloc
   - rowserrcheck
-  - scopelint
+  - exportloopref
```

Comment:
I think the linters are sorted alphabetically

Reply:
Do we have a linter for that linter 🙃 ? I got no errors locally, but I'd rather not enforce something that cannot be tested/validated

Reply:
Nope, but I guess not every review comment can be enforced by linter and the list would be nicer to navigate if it stays sorted :)

Reply:
Fixed it, although we should either automate that or just keep it random